PG2.NSP

```diff
@@ -0,0 +1,12 @@
+
+* >Natural Source Header 000000
+* :Mode S
+* :CP
+* <Natural Source Header
+/** New Program PGM1.
+/**
+/** :author Administrator
+/* TODO Enter your code here
+write 'WELCOME TO GIT HUB TRAINING'
+write 'day2'

```

Comment:
Adding comment to include new write statement for date

Reply:
Fixed